pkg/postgres/pgutils/error.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/jackc/pgx/v5/pgconn"
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/errorhelpers"
```

Comment:
Is it planned to move ErrorList to errox as well ?

Reply:
@dhaus67 has a PR to remove it altogether: https://github.com/stackrox/stackrox/pull/10099